labonneboite/importer/jobs/extract_etablissements.py

```diff
@@ -14,6 +14,9 @@
 from labonneboite.importer.jobs.common import logger
 
 
+def has_text_content(s):
+    return not(s is None or s == "" or s.isspace())
```

Comment:
`return s is not None and s.strip()`

Reply:
incorrect - `s.strip()` renvoit un string pas un booléen

`s.isspace()` renvoit `True` ssi `s` est constitué uniquement d'espaces

https://www.tutorialspoint.com/python/string_isspace.htm

Reply:
Oui, `s.trip()` renvoie une string : convertie en booléen, c'est False si la string est vide, et True sinon.

Reply:
Je cherchais juste à simplifier l'expression, qui me paraît alambiquée. Si l'évaluation en booléen n'est pas intuitive, on peut aussi écrire : 

    return s is not None and len(s) > 0 and not s.isspace()

Reply:
euh ok si tu y tiens tant que ça...

pinnaillage :D

finalement je mets

```
def has_text_content(s):
    is_empty = (s is None or s == "" or s.isspace())
    return not is_empty
```

Reply:
ok will do